apps/sensenet/src/components/field-controls/webhook-payload.tsx

```diff
@@ -106,35 +117,15 @@ export const WebhookPayload: React.FC<ReactClientFieldSetting<LongTextFieldSetti
     default:
       return (
         <>
-          <FormLabel style={{ transform: 'translate(0, 1.5px) scale(0.75)', transformOrigin: 'top left' }}>
+          <Typography variant="caption" gutterBottom={true}>
             {localization.webhooksPayload.payload}
-          </FormLabel>
-          <RadioGroup
-            aria-label="payload"
-            name="payload"
-            value={String(useDefault)}
-            onChange={(event) => setUseDefault(event.target.value === 'true')}>
-            <FormControlLabel
-              disabled
-              value="true"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.useDefault}
-            />
-            <FormControlLabel
-              disabled
-              value="false"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.customize}
-            />
-          </RadioGroup>
-          <TextareaAutosize
-            className={classes.textArea}
-            disabled
-            aria-label="minimum height"
-            rowsMin={10}
-            cols={70}
-            value={value}
-          />
+          </Typography>
+          <Typography variant="body1" gutterBottom={true}>
+            {String(useDefault) ? localization.webhooksPayload.defaultPayload : localization.webhooksPayload.customize}
```

Comment:
useDefault is a boolean, so it should be: (otherwise it is not working for me)
```suggestion
            {useDefault ? localization.webhooksPayload.defaultPayload : localization.webhooksPayload.customize}
```